### DISCUSSION TITLE
[Enhancement] First class support for Cupertino and Fluent Visual Renderer

### DISCUSSION THREAD

**Discussion by sps014** [dotnet/maui#168] [General]
Xamarin is not able to create more renderers for forms, only Material visual Renderer is supported.
It is necessary to support Cupertino and Fluent Design Visual Renderer in MAUI.

**Comment by MBuelowius** [comment]
Cupertino renderers are kinda problematic because because the Fonts are not allowed on different platforms. Google addressed this problem in Flutter with a different font. Fluent on the other hand should be definitely be supported because Microsoft owns it.

**Comment by DamienDoumer** [comment]
I really really really hope the support for Fluent design will be taken into consideration. It looks great and Microsoft should permit us to leverage the great visual language they created to build great apps with MAUI.

**Comment by sps014** [comment]
Agree Fluent Design Controls even on Office Mobile App looks cool.
It will be a bad idea to neglect these Visual Renderers in MAUI. 
Also One strange thing I observed from MS that they support fluent controls on React but they have never shown any interest to develop Blazor Fluent Component.
![image](https://user-images.githubusercontent.com/45932883/86514465-03390e80-be30-11ea-94f1-40d2f9b8ca1f.png)

[Image Source](https://www.theverge.com/2019/12/5/20996748/microsoft-fluent-design-mobile-office-apps-new-updates-features)
